### DISCUSSION TITLE
Stuck half updated to 6.4.0

### DISCUSSION THREAD

**Discussion by Cattrah** [YetiForceCompany/YetiForceCRM#16435] [General]
Ran into an error when updating from 6.3 to 6.4, tried to run the updater again and it "finished" but said there were errors so check the log. I checked the log and found the same error. Tried to run the updater again and now it's trying to install package 4 but shows that package 3 was not complete. Trying to install package 4 and it tells me that the default language is not there. Not sure how to proceed to get out of this jam. The program seems to be functioning fine, but there must be some underlying problems... I think I need to run package 3 update again without errors before I can run 4 successfully. I'm not sure how I can do that from the front end. This was the original error I had

YetiForceUpdate::roundcubeUpdateTable | 2022-09-04 17:21:57
SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes
The SQL being executed was: ALTER TABLE `roundcube_cache_index` CHANGE `mailbox` `mailbox` varchar(255)  COLLATE utf8mb4_bin NOT NULL after `user_id`, ROW_FORMAT=DYNAMIC;|#0 

What my updates screen looks like

![image](https://user-images.githubusercontent.com/101741600/188336556-99cb9796-3958-4df0-99ea-7d3b457f0d22.png)

after downloading package 4 and try to install
![image](https://user-images.githubusercontent.com/101741600/188336602-0c404dff-be04-4601-8fb7-fef2d919d325.png)

trying to update english language and I get a pop up error parsing server responses (I don't see any verbose text in any log files to know what the actual error is)
![image](https://user-images.githubusercontent.com/101741600/188336653-5efaa637-f967-4374-b52b-ddacfffda366.png)

Unfortunately, I think my yeti backup failed before updating so I'm not sure I can roll everything back so I'm trying to get out of this mess if I can.

**Comment by rskrzypczak** [comment]
Ideally, you should be able to restore the system from a backup, but if this is not possible, correct the database configuration, because you probably have a problem here.
![image](https://user-images.githubusercontent.com/10221999/188361076-8a840506-9610-47f2-87b4-327a38d6b90d.png)
Make a backup
Correct the server configurations _Software configuration/Logs/Server configuration_ (https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/web-server-requirements) 
Try to update to 6.4 again (previously changing the application version number to 6.3) https://yetiforce.com/pl/component/zoo/item/system-update
